### PR TITLE
updating form theming

### PR DESCRIPTION
**Pull request by katlazarova**

### CODE REVIEW

File: web/themes/custom/pineapple/sass/base/forms/_form-items.scss [hunk 1]
```diff
@@ -44,7 +51,21 @@ form {
...
+    margin-right: $unit;
+  }
+
   .fieldset-legend {
     font-size: 20px;
   }
+
+  .webform-button--submit.button {
+    background-color: white;
+  }
```
Comment (ElXyfer) [inline web/themes/custom/pineapple/sass/base/forms/_form-items.scss:70]: What's this for?

Comment (katlazarova) [reply web/themes/custom/pineapple/sass/base/forms/_form-items.scss:70]: on mobile, the submit button has a gray background

Comment (ElXyfer) [reply web/themes/custom/pineapple/sass/base/forms/_form-items.scss:70]: Why only on mobile?

Comment (katlazarova) [reply web/themes/custom/pineapple/sass/base/forms/_form-items.scss:70]: I have no clue what causes it.

**Comment by ElXyfer** [comment]
So some general comments to the entire pull request are as follows:
- the font sizes are too small i think you should remove all instances of you setting the font sizes in your form sass files and let them inherit the default font sizes of your site. ( but leave the descriptions the size they are )
- Once you do, if i were you I'd change the form labels and fieldset titles to have black text since that's the colour the rest of your site has 
- I would also look into slightly increasing the spacing between the letters because as they currently are they're a bit hard to read and i think increasing the spacing of the letters may help

**Comment by katlazarova** [comment]
Thank you for your suggestions. I will implement and get back to you.

File: web/themes/custom/pineapple/sass/base/forms/_form-items.scss (hunk 1 above)
Comment (ElXyfer) [reply web/themes/custom/pineapple/sass/base/forms/_form-items.scss:70]: You'll have to show me, what's going on.

**Pull request merged**